cms/djangoapps/contentstore/management/commands/tests/test_reindex_courses.py

```diff
@@ -116,3 +124,13 @@ def test_given_all_key_prompts_and_reindexes_all_courses_cancelled(self):
 
                 patched_yes_no.assert_called_once_with(ReindexCommand.CONFIRMATION_PROMPT, default='no')
                 patched_index.assert_not_called()
+
+    def test_given_active_key_prompt(self):
+        """ Test that reindexes all courses when --active key is given """
```

Comment:
we should probably test that it indexes only some of the courses by making one of the courses inactive

although below it only sets up first course so is that what the test is doing and the comment is wrong?